### PR TITLE
Nick dotnet 6 webapi post

### PR DESCRIPTION
**Pull request by Quorralyne**
This blog post has the following features:

- [x] A GitHub Repository with a polished README
- [x] A GitHub Repository under the github.com/oktadev account
- [x] A title that's approved by Dev Advocacy
- [x] A URL that's approved by Dev Advocacy
- [x] The content has been run through Grammarly (https://www.grammarly.com/)
- [x] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_posts/2022-04-20-dotnet-6-web-api.md [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+
+## Set up your Okta authorization server
+
+The first thing you'll need to do is set up your authorization server. Typically in a production environment, you would want to create a new authorization server, but the *default* server that comes with your Okta developer account should be sufficient for this demo. Navigate to the *API* section on the *Security* menu in your Okta admin portal.
+
+Note the *Audience* and *Issuer URI* of the `default` authorization server. You will need these in your application. Next, click on **default** and navigate to the *Scopes* tab. Add a new custom scope by clicking on **Add Scope** and give it the name `api`. Your `Display phrase` and `Description` can be anything you like.
+
+{% img blog/dotnet-6-webapi/image2.png alt:"add API scope" width:"600" %}{: .center-image }
+
+Next, navigate to the *Applications* page under the *Applications* menu. Click **Create App Integration** and then select *API Services*. Give your application a meaningful name; I named mine ``. Make a note of your `Client ID` and your `Client secret`. You will need these to obtain the access code from your authorization server.
```
Comment (alisaduncan) [inline _source/_posts/2022-04-20-dotnet-6-web-api.md:45]: > Give your application a meaningful name; I named mine ``.

Either missing the name or I'm slow at picking up sarcasm today

File: _source/_posts/2022-04-20-dotnet-6-web-api.md [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+
+## Create your web API
+
+Next, you can begin work on your resource API. Open Visual Studio 2022 and select *Create a new project*. Find the option for `ASP.NET Core Web API` and press **Next**.
+
+{% img blog/dotnet-6-webapi/image3.png alt:"create a new project" width:"800" %}{: .center-image }
```
Comment (alisaduncan) [inline _source/_posts/2022-04-20-dotnet-6-web-api.md:51]: This picture is pretty fuzzy. I wonder if the screenshot can be updated or maybe use a different width than 800?

File: _source/_posts/2022-04-20-dotnet-6-web-api.md [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+
+Name your application `Okta_ClientFlowDotNetSix` and click **Next**. Finally, under *Additional Information* select `.NET 6.0 (Long-term support)` as your framework. Click on **Create** and give VS a moment to scaffold your application.
+
+{% img blog/dotnet-6-webapi/image4.png alt:"additional information" width:"800" %}{: .center-image }
```
Comment (alisaduncan) [inline _source/_posts/2022-04-20-dotnet-6-web-api.md:55]: This picture is pretty fuzzy. I wonder if the screenshot can be updated or maybe use a different width than 800?

File: _source/_posts/2022-04-20-dotnet-6-web-api.md [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+    }
+  },
+  "AllowedHosts": "*",
+  "Okta": {
+    "Issuer": "{yourOktaDomain}/oauth2/default"
+  }
+}
+```
+
+The `default` endpoint in this URI refers to the authorization server. Assuming you used the `default` authorization server this should work.However, if you are using a custom authorization server you will want to obtain the `Issuer` from your Okta admin portal.
```
Comment (alisaduncan) [inline _source/_posts/2022-04-20-dotnet-6-web-api.md]:
```suggestion
The `default` endpoint in this URI refers to the authorization server. Assuming you used the `default` authorization server this should work. However, if you are using a custom authorization server you will want to obtain the `Issuer` from your Okta admin portal.
```

**Review by alisaduncan — COMMENTED** [review]
Good read. I added a couple of comments.

File: _source/_posts/2022-04-20-dotnet-6-web-api.md (hunk 2 above)
Comment (Quorralyne) [reply _source/_posts/2022-04-20-dotnet-6-web-api.md:51]: This is an issue Matt has been mentioning for a while. Not sure why this is happening, the images themselves are clear.

**Pull request merged**